### ISSUE TITLE
add more exercises to probability nb

### ISSUE THREAD

**Issue by charlesfrye** [wandb/edu#37]
Ideas:

1. **PMFs and PDFs**. Dictionaries versus densities.
1. **Softmax**. Define a softmax function for a PMF.
1. **Cross entropy loss**. Implement it, based on a formula? Based on PMF or PDF?
